### DISCUSSION TITLE
Cell type specific TF-gene GRN for multiome PBMC cell type CD14+ and FCGR3A+ Monocytes

### DISCUSSION THREAD

**Discussion by elhaam** [aertslab/scenicplus#318] [Q&A]
Hi Seppe and community,

I'm interested in obtaining the GRN inference tables for two PBMC cell types within the 10x multiome PBMC tutorial: CD14+ Monocytes and FCGR3A+ Monocytes. 

I saw cell type-specific mapping in the heatmap-dotplot section and the network plotting section visualizations. I was wondering what approach you suggest to extract cell type CD14+ and FCGR3A+ Monocytes GRNS? Should I follow the dot plots? 
Based on SCENIC+ pipeline, is it an effective approach to keep the cells in CD14+ and FCGR3A+ Monocyte clusters, filter out other cells from scRNAseq and scATACseq, and re-run SCENIC+ based on the PBMC tutorial to get the GRN only for Monocytes?

Similar to the [previous discussion #126 ](https://github.com/aertslab/scenicplus/discussions/126), I aim to generate tables for each cell type, listing TFname, Gene_name, and T2G_score.

Could you provide guidance or an example within the 10x multiome PBMC tutorial context? Additionally, considering the close relationship between CD14+ and FCGR3A+ Monocytes, do you recommend analyzing their GRNs together or separately?

Thanks,
Elham

**Comment by SeppeDeWinter** [comment]
Hi @elhaam 

Like I answered in https://github.com/aertslab/scenicplus/discussions/126, getting GRNs per cell type is not a trivial task. There is no _one_ way to do this.

Regarding your question wether you should subset the data to the cell types of interest or not. I always recommend to not do any such subsetting. You need the other cell types as a background to see what is specific to CD14+ and FCGR3A+ Monocyte cells. If you are interested in analysing the difference between these two cell types in particular, but don't care that much about what makes them specific in the context of all the other cell types you could consider subsetting these cell types and running SCENIC+ only on them.

I hope this answers your question?

All the best and good luck with your analyses. 
Seppe
